tests/helper.js

```diff
@@ -184,4 +185,29 @@
   };
 
 
+  exports.ReceiptValidator = function ReceiptValidator(opt) {
+    opt = utils.defaults(opt, {
+      response: {status: 'ok'},
+      onRequest: function() {},
+      verifyUrl: new RegExp(
+        'https://receiptcheck-payments-alt\\.allizom\\.org/verify/'),
```

Comment:
I suppose the '\\' (due to it being a string) is a price worth paying to save on needing to escape all the '/'.